api/base/serializers.py

```diff
@@ -1788,3 +1789,34 @@ class MaintenanceStateSerializer(ser.ModelSerializer):
     class Meta:
         model = MaintenanceState
         fields = ('level', 'message', 'start', 'end')
+
+
+class BaseProfileSerializer(JSONAPISerializer):
```

Comment:
The naming of these "base" classes was tricky - ended up using "Profile" for a lot of things that were education and employment related. Feel free to rethink and rename.